api/prometheus.go

```diff
@@ -113,134 +211,176 @@ func (hs HostSettings) PrometheusMetric() []prometheus.Metric {
-			Timestamp: time.Now(),
+			Name:  "hostd_metrics_contracts_successful",
+			Value: float64(m.Contracts.Successful),
 		},
 		{
-			Name:      "hostd_metrics_contracts_risked_collateral",
-			Value:     m.Contracts.RiskedCollateral.Siacoins(),
-			Timestamp: time.Now(),
+			Name:  "hostd_metrics_contracts_locked_collateral",
+			Value: m.Contracts.LockedCollateral.Siacoins(),
 		},
 		{
-			Name:      "hostd_storage_total_bytes",
-			Value:     float64(m.Storage.TotalSectors * rhp2.SectorSize),
-			Timestamp: time.Now(),
+			Name:  "hostd_metrics_contracts_risked_collateral",
+			Value: m.Contracts.RiskedCollateral.Siacoins(),
 		},
 		{
-			Name:      "hostd_storage_physical_used_bytes",
-			Value:     float64(m.Storage.PhysicalSectors * rhp2.SectorSize),
-			Timestamp: time.Now(),
+			Name:  "hostd_metrics_storage_total_bytes",
+			Value: float64(m.Storage.TotalSectors * rhp2.SectorSize),
 		},
 		{
-			Name:      "hostd_storage_contract_used_bytes",
-			Value:     float64(m.Storage.ContractSectors * rhp2.SectorSize),
-			Timestamp: time.Now(),
+			Name:  "hostd_metrics_storage_physical_used_bytes",
+			Value: float64(m.Storage.PhysicalSectors * rhp2.SectorSize),
 		},
 		{
-			Name:      "hostd_storage_temp_used_bytes",
-			Value:     float64(m.Storage.TempSectors * rhp2.SectorSize),
-			Timestamp: time.Now(),
+			Name:  "hostd_metrics_storage_contract_used_bytes",
+			Value: float64(m.Storage.ContractSectors * rhp2.SectorSize),
 		},
 		{
-			Name:      "hostd_storage_reads",
-			Value:     float64(m.Storage.Reads),
-			Timestamp: time.Now(),
+			Name:  "hostd_metrics_storage_temp_used_bytes",
+			Value: float64(m.Storage.TempSectors * rhp2.SectorSize),
 		},
 		{
-			Name:      "hostd_storage_writes",
-			Value:     float64(m.Storage.Writes),
-			Timestamp: time.Now(),
+			Name:  "hostd_metrics_storage_reads",
+			Value: float64(m.Storage.Reads),
 		},
 		{
-			Name:      "hostd_data_rhp_ingress_bytes",
-			Value:     float64(m.Data.RHP.Ingress),
-			Timestamp: time.Now(),
+			Name:  "hostd_metrics_storage_writes",
+			Value: float64(m.Storage.Writes),
 		},
 		{
-			Name:      "hostd_data_rhp_egress_bytes",
-			Value:     float64(m.Data.RHP.Egress),
-			Timestamp: time.Now(),
+			Name:  "hostd_metrics_storage_sector_cache_hits",
+			Value: float64(m.Storage.SectorCacheHits),
 		},
 		{
-			Name:      "hostd_wallet_balance",
-			Value:     m.Balance.Siacoins(),
-			Timestamp: time.Now(),
+			Name:  "hostd_metrics_storage_sector_cache_misses",
+			Value: float64(m.Storage.SectorCacheMisses),
+		},
+		{
+			Name:  "hostd_metrics_registry_entries",
+			Value: float64(m.Registry.Entries),
+		},
+		{
+			Name:  "hostd_metrics_registry_max_entries",
+			Value: float64(m.Registry.MaxEntries),
+		},
+		{
+			Name:  "hostd_metrics_registry_reads",
+			Value: float64(m.Registry.Reads),
+		},
+		{
+			Name:  "hostd_metrics_registry_writes",
+			Value: float64(m.Registry.Writes),
+		},
+		{
+			Name:  "hostd_metrics_data_rhp3_ingress",
```

Comment:
`rhp` was correct

Reply:
this one took me a moment to figure out what you meant by that. updated in next commit